src/main/java/org/moltimate/moltimatebackend/util/PdbXmlClient.java

```diff
@@ -1,107 +1,141 @@
 package org.moltimate.moltimatebackend.util;
 
 import lombok.extern.slf4j.Slf4j;
+import org.json.simple.JSONArray;
+import org.json.simple.JSONObject;
+import org.json.simple.parser.JSONParser;
+import org.json.simple.parser.ParseException;
 
-import java.io.BufferedReader;
-import java.io.IOException;
-import java.io.InputStream;
-import java.io.InputStreamReader;
-import java.io.OutputStreamWriter;
+import java.io.*;
+import java.net.HttpURLConnection;
 import java.net.URL;
-import java.net.URLConnection;
 import java.net.URLEncoder;
+import java.nio.charset.StandardCharsets;
 import java.util.ArrayList;
+import java.util.Iterator;
 import java.util.List;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
-import java.util.stream.Collectors;
 
 @Slf4j
 public class PdbXmlClient {
 
-    private static final String PDB_POST_SEARCH_URL = "http://www.rcsb.org/pdb/rest/search";
-    private static final String PDB_GET_CURRENT_PDB_IDS_URL = "https://www.rcsb.org/pdb/rest/getCurrent";
+    private static final String PDB_GET_CURRENT_PDB_IDS_URL = "https://data.rcsb.org/rest/v1/holdings/current/entry_ids";
 
+    //get the pdb ids that are in the same ec class
     public static List<String> postEcNumberForPdbIds(String ecNumber) {
-        String xml = "<orgPdbQuery>" +
-                "<queryType>org.pdb.query.simple.EntriesOfEntitiesQuery</queryType>" +
-                "<description>Entries of :EnzymeClassificationQuery Search for " + ecNumber + "</description>" +
-                "<parent><![CDATA[<orgPdbQuery>" +
-                "<queryType>org.pdb.query.simple.EnzymeClassificationQuery</queryType>" +
-                "<description>Enzyme Classification Search : EC=" + ecNumber + "</description>" +
-                "<Enzyme_Classification>" + ecNumber + "</Enzyme_Classification>" +
-                "</orgPdbQuery>]]></parent>" +
-                "</orgPdbQuery>";
+        log.info("Post EC number for PDB Ids");
+        List<String> pdbIds = new ArrayList<>();
+
+        String USER_AGENT = "Mozilla/5.0";
```

Comment:
This is the second time this has been used; maybe put it in one of our Util files.